comment/tests/test_api.py

```diff
@@ -504,6 +504,13 @@ def test_change_state_when_flagging_is_disabled(self):
         comment = self.comment_2
         self.assertFalse(comment.is_flagged)
         response = self.client.post(self.get_url(), data=self.data)
+        self.assertEqual(response.status_code, 403)
+        
+    @patch.object(settings, 'COMMENT_FLAGS_ALLOWED', 1)
+    def test_change_state_when_comment_is_not_flagged(self):
+        comment = self.comment_2
+        self.assertFalse(comment.is_flagged)
```

Comment:
These two lines are required inside this function as now. You may want to remove them,